main.go

```diff
@@ -66,20 +66,20 @@ func init() { flag.Parse() }
 
 // Main function of the bot, called on startup.
 func main() {
-	// Load environment variables
-	getEnv := func(key string) {
-        val, ok := os.LookupEnv(key)
-        if !ok {
-            fmt.Printf("%s is not present, please check your .env file.\n", key)
-			// stop further execution of the program 
-			os.Exit() 
-        } else {
-            return val
-        }
-    }
+	//load env variables
+	err := godotenv.Load(".env")
+	if err != nil {
+		log.Fatalf("Error loading .env file")
+		return
+	}
+	//apply env variables to secrets
+	token = os.Getenv("TOKEN")
+	aptly = os.Getenv("API_KEY")
+	logChannelID = os.Getenv("LOG_CHANNEL_ID")
+	botPrefix = os.Getenv("BOTPREFIX")
```

Comment:
Small change, there needs to be an underscore here: `BOT_PREFIX`, as it is specified in that way in the `.env.sample` file.